static/app/views/alerts/create.tsx

```diff
@@ -126,7 +126,8 @@ function Create(props: Props) {
       : 'issues';
   }
 
-  const title = t('New Alert Rule');
+  const title =
+    alertType === AlertRuleType.UPTIME ? t('Create Uptime Monitor') : t('New Alert Rule');
```

Comment:
I think what I would prefer to see would be a module level variable named like

```tsx
const TITLES: Record<AlertRuleType, React.ReactNode> = {
  [AlertRuleType.UPTIME]:  t('Create Uptime Monitor'),
}
```

then this line becomes

```suggestion
  const title = TITLES[alertType] ?? t('New Alert Rule');
```